develop/tutorials/articles/280-workflow/01-creating-workflow-definitions/03-notifications.markdown

```diff
@@ -48,38 +55,56 @@ processing is leaving a node (`onExit`). If you specify a notification to
 be sent on assignment, the assignee is automatically notified.
 
 **Recipients**
-: Decide who should receive the notification in the `<recipients>` tag.
+: Decide who should receive the notification in the `<recipients>` tag:
+
+        <recipients>
+            [SEE BELOW FOR THE AVAILABLE RECIPIENT TAGS]
+        </recipients>
+
 Available recipient tags are 
 
-- `<user />`: notify the User that sent the asset through the workflow using.
+- `<user>`: notify the User that sent the asset through the workflow. Simply
+    specify the tag as `<user />`. To notify a specific user, enter the
+    `userId`:
+
+        <recipients>
```

Comment:
Hey @rbohl, unfortunately, we can't refer to two different recipients types at once, like this XML. The current implementation only accepts _Asset creator_ and _User_ separately.